src/hooks/use-location-sync.js

```diff
@@ -0,0 +1,92 @@
+import { createBrowserHistory } from 'history';
+import QS from 'query-string';
+import isEqual from 'react-fast-compare';
+import merge from 'deepmerge';
+
+// Needed to properly merge query strings. (Hook consumers can also provide
+// their own merge function if necessary)
+function defaultMergeLocation(object, ...sources) {
+  return mergeWith(object, ...sources, (left, right, key) => {
```

Comment:
I don't see `mergeWith` anywhere in scope here?

Reply:
It's just a named curried function so 
`mergeLocation = defaultMergeLocation,` gets set in the function header

Then 
```
const nextLocation = mergeLocation(
        {},
        location,
        mapStateToLocation(initialState)
      );
      history.replace(nextLocation);
```
Tells me it just gets called as it is returned. I guess we don't need to name it.

Reply:
I don't think I explained that right... 😉   I mean the exact characters `mergeWith` do not occur anywhere else within this file at https://github.com/FormidableLabs/spectacle/blob/7918632c58ef3482776f257da70666f7716f0388/src/hooks/use-location-sync.js

Reply:
Oooh you are right. We are missing that import from lodash. I will look into that before merging this.